Close #22. Also, made some addtional improvements to df2bib()

pegeler opened the pull request:
Hello,

I noticed that #22 was an easy win, so I made the change in the documentation for that.

While I was looking at the code, I noticed a few opportunities for extending the function, which I also implemented:

- The user now has the option to write to `stdout`.
- The user can now feed a `tibble` wherein there is a mix of `data.frames` and character vectors in the AUTHORS column. This might arise when the user is binding multiple `tibble`s together (hey, it could happen!!!).

I also took the liberty of updating the URLs to match the transition into the ropensci/bib2df repository, as well as naming myself as a contributor.

Hope that these changes meet with your approval!

Thanks,

Paul

ottlngr commented:
Thanks for your work and the improvements on `bib2df`! Just noticed that some unit tests fail, probably because I moved my webspace. I'll try to fix that and will host these .bib files as gists. Then I will thankfully merge your PR!

pegeler commented:
Hi @ottlngr!

Thanks for the great package and thanks for the feedback! 

I just saw the unit test issue and fixed it. I also updated the *.Rd* file, which I forgot to do in my original commit!
